cmake/external/abseil-cpp.cmake

```diff
@@ -50,96 +50,98 @@ endif()
 # TODO: since multiple ORT's dependencies depend on Abseil, the list below would vary from version to version.
 # We'd better to not manually manage the list.
 set(ABSEIL_LIBS
+absl::absl_check
```

Comment:
why log_internal_check_op should be added? Who depends it?

Reply:
see https://github.com/microsoft/onnxruntime/pull/22379#discussion_r1795737803

Reply:
not adding it now.